atlp-rwanda/bravo-bn-be
# #182201715 users registration with JWT

#### What does this PR do?
This is a user registration PR 

#### Description of Task to be completed?
- creating registration endpoint
- authentication controller

#### How should this be manually tested?
After running the app with `npm run start:dev` use this endpoint `/api/v1/user/signup` on a `POST` request to register a user.
 `GET` `/api/v1/user/` to view inserted user

-before you do anything make sure you have a testing database in your postgres. open a `.env` file and define `TEST_DATABASE=` add your database name and `NODE_ENV=test`
 Run `npm test`

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/182201715

#### Screenshots (if appropriate)
![Screenshot (236)](https://user-images.githubusercontent.com/59047760/170684474-760392c2-29d5-419b-9bf5-c664b74e34af.png)
